src/lib/changelog-parser.ts

```diff
@@ -7,7 +7,7 @@ import {escape as escapeRegex} from './utils/regex';
 export default class ChangelogParser {
   parse(changelog: string, knownVersion: Version): Changelog {
     const heading = this.findVersionHeading(changelog, knownVersion);
-    if (!heading) return createInvalidChangelog();
+    if (!heading) {return createInvalidChangelog();}
```

Comment:
Could we use `curly: ["error", "multi-line"]` to avoid adding curly braces here?

https://eslint.org/docs/latest/rules/curly#multi-line